### PR TITLE
Add test script as helper to build test resources

### PR DESCRIPTION
**Pull request by dr0i**

### CODE REVIEW

File: jsonDiffAndCopy.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#/bin/bash
+# This diffs the jsonld documents using the the cli and copy the jsonld
```
Comment (fsteeg) [inline jsonDiffAndCopy.sh]: Small typo here (`the` twice).

**Comment by fsteeg** [comment]
I don't quite understand in what context this is used. The comment suggest it's for running JUnit tests. In that case, shouldn't it be part of the test code? Is that what you mean with the TODO in the comment ('TODO Should be changed in the java sources, I guess.')?

**Comment by dr0i** [comment]
It's about rebuilding the test data when new transformation rules changes the data. This could (and should) be done in java, I will think about how to do it there.

**Comment by dr0i** [comment]
If you like implement it with java, would be nice.

**Comment by acka47** [comment]
@fsteeg You have this on the radar? Just asking because this is a lonely PR in the Ready column...

**Comment by dr0i** [comment]
Closed this in favor of #58.

**Pull request closed without merging**